packages/anthropic/src/anthropic-messages-language-model.ts

```diff
@@ -360,9 +352,9 @@ export class AnthropicMessagesLanguageModel implements LanguageModelV3 {
     } = await prepareTools(
       jsonResponseTool != null
         ? {
-            tools: [jsonResponseTool],
-            toolChoice: { type: 'tool', toolName: jsonResponseTool.name },
-            disableParallelToolUse: true,
+            tools: tools ? [...tools, jsonResponseTool] : [jsonResponseTool],
+            toolChoice: { type: 'required' },
+            disableParallelToolUse: anthropicOptions?.disableParallelToolUse,
```

Comment:
the setting was `true` on purpose so we can guarantee that only the json response tool is called in the last step, and only one time (not in parallel)